core/src/main/kotlin/gropius/repository/RepositoryExtensions.kt

```diff
@@ -12,7 +12,7 @@ import org.springframework.data.repository.reactive.ReactiveCrudRepository
  * @param id the id of the node to find
  * @return the found node
  */
-suspend fun <T> ReactiveNeo4jRepository<T, String>.findById(id: ID): T {
+suspend fun <T> GropiusRepository<T, String>.findById(id: ID): T {
```

Comment:
move into `GropiusRepository`

Reply:
sadly not possible, as in GropiusRepository, ID is a generic type, while in the extension function it is not